public/index.html

```diff
@@ -9,7 +9,7 @@
     <h1>Welcome to RuneHelper!</h1>
     <p>To check your stats, change the url:</p>
     <p>Example:</p>
-    <p>localhost:3000/INSERT_USERNAME_HERE</p>
+    <p>localhost:3000/search/INSERT_USERNAME_HERE</p>
```

Comment:
Is this file still used now that you have frontend/public/index.html?

Reply:
this folder is no longer being used at all.

deleted!